include/QBDI/VM.h

```diff
@@ -74,13 +111,13 @@ class QBDI_EXPORT VM {
      *
      * @param[in] gprState A structure containing the GPR state.
      */
-    void        setGPRState(GPRState* gprState);
+    void        setGPRState(const GPRState* gprState);
```

Comment:
It could be better to use a const-ref to avoid the user to provide a nullptr

Reply:
why not, but we should also change getGPRState to give a reference instead of a pointer. This change will also break the API.